.cargo/config

```diff
@@ -0,0 +1,5 @@
+# We are using UUID v7 which is in final comment period:
```

Comment:
What's the rationale for using UUIDv7 given that its still not fully ready?

Reply:
So there is this whole science about what uuid to use unfortunately :/:

https://blog.devgenius.io/analyzing-new-unique-identifier-formats-uuidv6-uuidv7-and-uuidv8-d6cc5cd7391a
https://vladmihalcea.com/uuid-database-primary-key/
https://brandur.org/nanoglyphs/026-ids

The summary seems to be that uuids as primary keys were a bad idea in the past because of the format, so some people advocated for using TSIDs... And it seems that UUIDv7 fixes most of those shortcomings (by being similar to TSIDs) of previous formats so the general consus seems to be to use UUIDv7 for databases now...

Re: Not fully ready: v7 has entered last call period 
e.g., https://github.com/uuid6/prototypes/issues/42#issuecomment-1542619709 

which says:
> Please comment on the tracker or email thread before June 9th to request any changes.

so I *hope* this file to not be needed for very long -- if it stalls for some reason and isn't ready before we release we can simply replace all `v7_now` calls with something else

Reply:
Okay! Maybe just fold in this summary somewhere. :)